code/__HELPERS/trait_helpers.dm

```diff
@@ -232,6 +232,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_TABLE_LEAP "table_leap" // Lets bartender and chef mount tables faster
 #define TRAIT_NEVER_MISSES_DISPOSALS "trait_never_misses_disposals" // For janitors landing disposal throws
 #define TRAIT_SLEIGHT_OF_HAND "sleight_of_hand"
+#define TRAIT_EXPERT_CHEF "expert_chef"
```

Comment:
```suggestion
#define TRAIT_EXPERT_CHEF "expert_chef"
```
These are purely backend so it's the standard to keep their names functional. maybe something like TRAIT_KNOWS_COOKING_RECIPE